src/NuGet.Clients/PackageManagement.VisualStudio/ProjectSystems/VSMSBuildNuGetProjectSystem.cs

```diff
@@ -951,5 +961,17 @@ public virtual IEnumerable<string> GetDirectories(string path)
                        select p.Name;
             });
         }
+
+        private static bool IsReferenceUnavailableException(Exception e)
+        {
+            var comException = e as COMException;
```

Comment:
This is a failure from the old .csproj project system? What will happen on the new CPS csproj?